bin/securedrop_server

```diff
@@ -15,4 +16,19 @@ if __name__ == "__main__":
         if not os.path.exists(securedrop_dir):
             raise RuntimeError("Failed to create securedrop directory: $HOME/.securedrop")
     securedrop_file = os.path.join(securedrop_dir, "server_db.json")
-    server.main(filename=securedrop_file)
+    securedrop_port = None
+    try:
+        opts, args = getopt.getopt(sys.argv[1:], "pfv", ["port", "filename", "verbose"])
+    except getopt.GetoptError as err:
+        print(err)  # will print something like "option -a not recognized"
+        sys.exit(2)
+    for o, a in opts:
+        if o in ("-v", "--verbose"):
+            VerbosePrinter(True)
```

Comment:
In the bin/securedrop file "VerbosePrinter(true)" is checked last in the if else chain. Here it is first. I think the logic here should match the bin/securedrop file.